apps/explorer/src/components/validator-map/ValidatorMap.tsx

```diff
@@ -29,23 +30,31 @@ const DATE_FILTER_TO_WINDOW = {
 
 export default function ValidatorMap() {
     const [dateFilter, setDateFilter] = useDateFilterState('D');
+    const [isFail, setIsFail] = useState(false);
```

Comment:
This is already tracked as an error state from `useQuery`, see docs here: https://tanstack.com/query/v4/docs/reference/useQuery

Reply:
I've implemented this refactoring